addons/mp5/CfgWeapons.hpp

```diff
@@ -0,0 +1,12 @@
+class CfgWeapons {
+    class Rifle_Base_F;
+    class CUP_smg_MP5SD6: Rifle_Base_F {
+        magazines[] = {
+            QCLASS(30Rnd_9x19_MP5_7N21),
+            QCLASS(30Rnd_9x19_MP5_7N21_Green)
+        };
+        magazineWell[] = {
+            QCLASS(9x19_MP5)
```

Comment:
This is what actually prevents the invisible magazines to be loaded, right?

Reply:
Yeah, changed it from the CBA_9x19_MP5 so it doesn't see the Vanilla ones anymore.